.changeset/healthy-windows-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-css': patch
+'@itwin/itwinui-react': patch
+---
+
+Fixed a Firefox-specific bug where focus outlines were not appearing correctly around menu items (e.g. inside `ComboBox` and `Select`).
```

Comment:
Was this issue also in `Select`? When I tested it on Mac + Firefox 121.0, the outlines seemed to show correctly:

https://github.com/iTwin/iTwinUI/assets/45748283/9c882699-5a07-4b94-ba68-36a443a60321

Reply:
I also can't reproduce it now, so I'll change the changelog

```suggestion
Fixed a Firefox-specific bug where focus outlines were not appearing correctly around menu items inside `ComboBox`.
```